setup.py

```diff
@@ -116,7 +116,7 @@
             # For render
             "opencv-python",
             # For atari games,
-            "atari_py==0.2.6",
+            "gym[atari,accept-rom-license]>=0.21",
```

Comment:
I think this doesn't work (I remember testing it in the past), we should put autorom with accept license here

Reply:
I just tested it and it seems to work OK:
```
virtualenv test_venv
. ./test_venv/bin/activate
pip install -e .[extra]
```
then:
```
$ python -c 'import gym; gym.make("BreakoutNoFrameSkip-v4")'
A.L.E: Arcade Learning Environment (version 0.7.4+069f8bd)
[Powered by Stella]
```

IIRC it fails on some very old pip versions that don't support backtracking. I tested on pip 20.3.4 and Python 3.9, but I'm pretty sure it works on older versions (there's some discussion about this in the review).